datasets/us/ss_wanted/us_ss_wanted.yml

```diff
@@ -0,0 +1,53 @@
+title: "United States Secret Service Most Wanted Fugitives"
+entry_point: crawler.py
+prefix: us-ss-wanted
+summary: "Wanted fugitivies listed on United States Secret Service website."
+description: >
+  This dataset publishes the details of the most wanted fugitives listed on the United States Secret Service website.
+
+  Individuals in this list are involved in one or more financial crime, including but not limited to counterfeiting, 
+  bank fraud, hacking, crypto and credit card fraud. 
+
+  The dataset includes details such as the fugitive's name, date of birth, aliases, summary of the criminal case, and relevant links to them.
+
+url: https://www.secretservice.gov/investigations/mostwanted
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage: monthly
+publisher:
+  name: United States Secret Service
+  acronym: USSS
+  description: |
+    The United States Secret Service (USSS) is a federal law enforcement agency under the Department of Homeland Security 
+    charged with conducting criminal investigations and protecting U.S. political leaders, their families, and visiting 
+    heads of state or government. [Source](https://en.wikipedia.org/wiki/United_States_Secret_Service)
+
+    They also protect the integrity of the U.S. currency and investigate crimes against the U.S. financial system committed by criminals 
+    around the world and in cyberspace.
+
+  country: us
+  url: https://www.secretservice.gov/
+  official: true
+data:
+  url: https://www.secretservice.gov/investigations/mostwanted
+  format: HTML
+lookups:
+  type.date:
+    lowercase: true
+    options:
+      - match:
+          - N/A
+        value: null
+assertions:
+  min:
+    schema_entities:
+      Person: 24
+    country_entities:
```

Comment:
I don't think we need country assertions in this case because if the list changes, say for instance one of the Russians is not on the list any more, it would fail. Also, the `schema_entities` assertions could be a little lower, see this part of the documentation https://zavod.opensanctions.org/metadata/#data-assertions.

Reply:
Country assertions are useful to guard against e.g. the country parsing failing. But I usually make it quite loose so that normal variation in the data doesn't incorrectly result in failure. I try to capture the shape of the data though - say assert 70% of the values for the top two countries, and one or two of the countries with fewer entities to capture breadth.

In this case I'd say put russia at 8, ukraine at 4, and moldova and the US at 1 and skip the rest. we want to balance maintenance effort with detecting failures